src/extension/kernel/kernel.ts

```diff
@@ -90,11 +90,11 @@ export class Kernel implements NotebookKernel {
             }
             task.appendOutput(new NotebookCellOutput(outputItems));
         } catch (ex) {
-            const error: Error = ex;
+            const error: Error | undefined = ex;
```

Comment:
wtF... how can u throw a `null` exception, that feels wrong on all levels..
Oh well.
Would never have thought of that.thanks

Reply:
<img width="262" alt="Screen Shot 2021-04-07 at 1 09 24 AM" src="https://user-images.githubusercontent.com/30305945/113832939-ef636580-973d-11eb-951b-a3b1ea7bc7c9.png">

Gotta love JavaScript. Apparently you can throw anything